frontends/infinite-corridor/src/pages/SearchPage.tsx

```diff
@@ -32,94 +24,43 @@ const facetMap: FacetManifest = [
   ["offered_by", "Offered By"]
 ]
 
-interface Result {
-  _source: LearningResourceSearchResult
-}
-
-const SEARCH_API_URL = "search/"
-
-const search = async (params: SearchQueryParams) => {
-  const body = buildSearchQuery(params)
-  try {
-    const { data } = await axios.post(SEARCH_API_URL, body)
-    return data
-  } catch (err) {
-    return null
-  }
-}
-
 const SearchPage: React.FC = () => {
-  const [results, setSearchResults] = useState<Result[]>([])
-  const [total, setTotal] = useState(0)
-  const [completedInitialLoad, setCompletedInitialLoad] = useState(false)
-  const [requestInFlight, setRequestInFlight] = useState(false)
-  const [searchApiFailed, setSearchApiFailed] = useState(false)
-  const [aggregations, setAggregations] = useState<Aggregations>(new Map())
```

Comment:
None of these are necessary anymore. React query handles things like `completedInitialLoad`, `requestInFlight`, `searchApiFailed` itself. The other three are individual pieces of the search response. React query stores the responses, and we'll compute what we care about with `useMemo`.